cmd/wallet-server/startcmd/start_test.go

```diff
@@ -162,26 +206,8 @@ func TestStartCmdWithMissingArg(t *testing.T) {
 	t.Run("test invalid auto accept flag", func(t *testing.T) {
```

Comment:
It looks to me like this test is a holdover from a command line parameter that no longer exists